index.html

```diff
@@ -5,6 +5,37 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <script
+      src="https://kit.fontawesome.com/1e5a35ab03.js"
+      crossorigin="anonymous"
+    ></script>
   </head>
-  <body></body>
+  <body>
+    <div class="container">
+      <!-- create button -->
+      <div class="create">
+        <h1>Add Tasks</h1>
+        <button id="createButton">
+          <i class="fa-duotone fa-plus"></i>Create
+        </button>
+      </div>
+
+      <!--user input interface-->
+      <div class="hide" id="inputTask">
+        <input type="text" id="input" placeholder="Add a Todo" /> <br />
+        <button id="addTaskButton">Add Task</button>
+        <div class="trashInput">
+          <img src="icons/delete.svg" job="delete" id="0" />
```

Comment:
What is this job attribute? If you want to have a unique identifier to catch this HTML element please try to use a unique ID and then catch it using that. If that's not possible please use data attributes.

Reply:
I am using the id attribute to have control over individual tasks. that's why I am updating custom job attributes with data attributes. job ==> data-job.